CMakeLists.txt

```diff
@@ -181,14 +181,14 @@ file(GLOB S2N_SRC
     ${UTILS_SRC}
 )
 
-add_library(${PROJECT_NAME} ${S2N_HEADERS} ${S2N_SRC})
+add_library(${PROJECT_NAME} STATIC ${S2N_HEADERS} ${S2N_SRC})
```

Comment:
I thought `STATIC` indicates we are building a static library - which we only sometimes do with s2n (depending on other configs). Butmy cmake knowledge is only very superficial, so it might actually mean something different.

Reply:
My reading indicates this forces to find the static library in the subsequent target_link_libraries.  There is a build option later in the file to enable the shared library build.